src/components/shared/Validations.tsx

```diff
@@ -18,49 +21,54 @@ export const Validations: React.FunctionComponent<IValidations> = ({ required, v
     </div>
   );
 
-  return validationCount ? (
-    <Popover
-      boundary="window"
-      interactionKind="hover"
-      content={
-        <div className="p-5" style={{ maxHeight: 500, maxWidth: 400 }}>
-          {Object.keys(validations).map((key, index) => {
-            const validation = validations[key];
+  return (
+    <>
+      {deprecated ? <span className="ml-2 text-darken-7 dark:text-lighten-6">deprecated</span> : null}
+      {validationCount ? (
+        <Popover
+          boundary="window"
+          interactionKind="hover"
+          content={
+            <div className="p-5" style={{ maxHeight: 500, maxWidth: 400 }}>
+              {Object.keys(validations).map((key, index) => {
+                const validation = validations[key];
 
-            let elem = null;
-            if (Array.isArray(validation)) {
-              elem = validation.map((v, i) => (
-                <div key={i} className="mt-1 mr-1 flex items-center">
-                  <div className="px-1 bg-gray-2 dark:bg-gray-8 font-bold text-sm rounded">{String(v)}</div>
-                  {i < validation.length - 1 ? <div>,</div> : null}
-                </div>
-              ));
-            } else if (typeof validation === 'object') {
-              elem = (
-                <div className="m-1 px-1 bg-gray-2 dark:bg-gray-8 font-bold text-sm rounded" key={index}>
-                  {'{...}'}
-                </div>
-              );
-            } else {
-              elem = (
-                <div className="m-1 px-1 bg-gray-2 dark:bg-gray-8 font-bold text-sm rounded" key={index}>
-                  {JSON.stringify(validation)}
-                </div>
-              );
-            }
+                let elem = null;
+                if (Array.isArray(validation)) {
+                  elem = validation.map((v, i) => (
+                    <div key={i} className="mt-1 mr-1 flex items-center">
+                      <div className="px-1 bg-gray-2 dark:bg-gray-8 font-bold text-sm rounded">{String(v)}</div>
+                      {i < validation.length - 1 ? <div>,</div> : null}
+                    </div>
+                  ));
+                } else if (typeof validation === 'object') {
+                  elem = (
+                    <div className="m-1 px-1 bg-gray-2 dark:bg-gray-8 font-bold text-sm rounded" key={index}>
+                      {'{...}'}
+                    </div>
+                  );
+                } else {
+                  elem = (
+                    <div className="m-1 px-1 bg-gray-2 dark:bg-gray-8 font-bold text-sm rounded" key={index}>
+                      {JSON.stringify(validation)}
```

Comment:
Should we safe stringify here?

Reply:
I don't know. I'd say no, since you won't be able to recognize strings and numbers, i.e. "1" and 1.

Example:
- JSON.stringify
  - string 2
    ![image](https://user-images.githubusercontent.com/9273484/74425199-e0f52300-4e85-11ea-8e05-f3adbe264a18.png)
  - number 2
![image](https://user-images.githubusercontent.com/9273484/74425238-f36f5c80-4e85-11ea-8944-c6f52c9df5ee.png)

- safe stringify
  - string 2
![image](https://user-images.githubusercontent.com/9273484/74425096-b60acf00-4e85-11ea-8dbc-d897f5f9ffcf.png)
  - number 2
![image](https://user-images.githubusercontent.com/9273484/74425128-c0c56400-4e85-11ea-94e3-e552e722cb48.png)